### PR TITLE
feat: add docs for rate-limiter

### PR DESCRIPTION
**Pull request by pratik151192**
Added a `How To` section and added rate-limiter instructions to it. This was tested against the snippet here https://github.com/momentohq/client-sdk-javascript/pull/1002/files locally by switching branches. Note that we have an issue to support Tabs when we are injecting entire files https://github.com/momentohq/public-dev-docs/issues/506

### CODE REVIEW

**Comment by pratik151192** [comment]
The deployment will fail right now as the other PR isn't merged https://github.com/momentohq/client-sdk-javascript/pull/1002/files. Will re-run the check once that's merged

File: docs/cache/how-to/rate-limiter.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+The rate limiter increments the value of the unique key for each user when they make a request, setting a time-to-live (TTL) for the first request of the minute to 60 seconds. This is important as we want our keys to expire as they are not meaningful after they have served their purpose for a given minute.
+
+A flow of the rate-limiter looks like:
+
+- Increment the value of `user_id-current_minute`. If the returned value is 1, that indicates that this was the first request for the user for that given minute. Note that Momento's `increment` API is guaranteed to be atomic. If this return value is 1, we set the TTL of that key using `updateTTL` API to be 60 seconds.
+- If the value is less than the configured TPM limit for the rate limiter, we allow the request, or else, throttle it.
+
+Let's dive right into our implementation; pay attention to comments in this code where we explain the thought process.
+
+<SdkExampleCodeBlock language={'javascript'} file={'examples/nodejs/rate-limiter/doc-examples-files/doc-examples.ts'} />
```
Comment (pratik151192) [inline docs/cache/how-to/rate-limiter.mdx:47]: We need to plug tabs here. Not blocking right now but will be when we plan to add a new language. I played around the code and need some React knowledge to successfully get it out so will do that on my own time over the next few days/week https://github.com/momentohq/public-dev-docs/issues/506 have assigned myself to it.

Comment (allenheltondev) [reply docs/cache/how-to/rate-limiter.mdx:47]: You can use the `SdkExampleTabs` component here @pratik151192. Take a look at `./docs/cache/develop/api-reference/index.mdx`

Comment (pratik151192) [reply docs/cache/how-to/rate-limiter.mdx:47]: yea that  just supports functions so we need to add some react stuff to support files + tabs https://gomomento.slack.com/archives/C055MCGD2CD/p1698969486257659

File: docs/cache/how-to/_category_.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "label": "How To",
+    "position": 4,
+    "collapsible": true,
+    "collapsed": true,
+    "link": {
+        "description": "Things you can do with Momento."
```
Comment (allenheltondev) [inline docs/cache/how-to/_category_.json]: I'd say "things you can build with Momento" instead of "do"

**Review by cprice404 — APPROVED** [review]

**Pull request merged**